libs/wallet/src/rest-connector-form.tsx

```diff
@@ -59,7 +59,7 @@ export function RestConnectorForm({
           autoFocus={true}
         />
         {errors.wallet?.message && (
-          <InputError intent="danger" className="mt-4">
+          <InputError data-testid="error" intent="danger" className="mt-4">
```

Comment:
can we prefix this with the component name or similar, bit too generic IMO

Reply:
Changed id to reflect component